src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1866,17 +1849,11 @@ void ApplicationManagerImpl::OnStreamingConfigured(
       sync_primitives::AutoLock lock(navi_service_status_lock_);
 
       NaviServiceStatusMap::iterator it = navi_service_status_.find(app_id);
-      if (navi_service_status_.end() == it) {
+      if (navi_service_status_.end() == it && !result) {
```

Comment:
@IGapchuk  not obvious what is result variable about. Please rename variable. Give more narative name